my_list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] Please 🙏🏽 implement `#each` method that yields all members of `@list` in succession. Kindly check this [article](https://medium.com/rubycademy/the-yield-keyword-603a850b8921) about yield.